service/debugger/debugger.go

```diff
@@ -729,12 +751,18 @@ func (d *Debugger) ClearBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoint
 	defer d.targetMutex.Unlock()
 
 	var bps []*proc.Breakpoint
+	var dbps []*api.Breakpoint
```

Comment:
Because of how this works you can just add this at the start and leave the rest as it is:

```
if bp, ok := d.disabledBreakpoints[requestedBp.Addr]; ok {
    delete(d.disabledBreakpoints, addr)
    return bp, nil
}
```